pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/SubscribeRateLimiter.java

```diff
@@ -140,6 +140,9 @@ private synchronized void updateSubscribeRate(ConsumerIdentifier consumerIdentif
     }
 
     public void onSubscribeRateUpdate(SubscribeRate subscribeRate) {
+        if (this.subscribeRate.equals(subscribeRate)) {
```

Comment:
Could `this.subscribeRate` be null?

Reply:
`this.subscribeRate` can not be null. Because it is initialized in the `SubscribeRateLimiter` constructor, and it's value is setted from `HierarchyTopicPolicies`, which at least the broker-level value is not null.